src/Files.App/Services/Windows/WindowsIniService.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Runtime.InteropServices;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+
+namespace Files.App.Services
+{
+	/// <inheritdoc cref="IWindowsIniService"/>
+	public sealed class WindowsIniService : IWindowsIniService
+	{
+		/// <inheritdoc/>
+		public int GetValue(string filePath, string section = "", string name = "", out string value)
```

Comment:
The limitation with this method is that getting multiple values will parse the file multiple times. It would be preferable to first parse the file, and have a separate GetValue method using the already parsed data.

Reply:
We should create our own data type containing what we want and return once per folder.

DesktopIniDataItem
- FolderBackgroundImage
- FolderType
- ViewState

Reply:
@yaira2 what do you think?